### DISCUSSION TITLE
Collection description visibility

### DISCUSSION THREAD

**Discussion by darioguarascio** [directus/directus#17813] [Feature Requests]
### Summary

Hi guys,

heavy directus user here. I still have a 9.0.0-rc.88 instance running, and in this version there is a "index" page for collections, that looks like have been lost over time.

In this index page you would see, along with the collection name, also its description, that, afaik now is only readable in Data Model, on mouse hovering the collection. Looks kinda lost to me.

Another excellent place to display this bit of information would be in the Collection page, as subtext of the collection name itself 

I think having a snippet of text to remind users what the collection is about is as useful as a Field Note

### Motivation

Having the collection description being more visible

### Detailed Design

In Collection page, title section

```
Content
**Collection Name**
<description>
```
or

```
Content
**Collection Name** - <description>
```

### Requirements List

No requirements, data is already there, it's only a display issue

### Drawbacks

If you think overloading the UI is a bad idea. In any case, it would only show if description is available

### Alternatives

Bring back the collection index page?

### Adoption Strategy

I imagine that, in Collection page, you already have collection metadata available, so adding the description should not be a big one

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
